Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1235,14 +1235,14 @@ def lhegensim(fragment,howMuch):
 
 # step4
 step4Defaults = { 
-                  '-s'            : 'ALCA:TkAlMuonIsolated+TkAlMinBias+MuAlOverlaps+EcalESAlign',
+                  '-s'            : 'ALCA:TkAlMuonIsolated+TkAlMinBias+MuAlOverlaps+EcalESAlign+EcalTrg',
                   '-n'            : 1000,
```

Comment:
I think we can remove it for the moment. I don't think we really need it on MC now.
What would be faster in your opinion?